framework/src/modules/http_api/controllers/blocks.js

```diff
@@ -40,52 +39,31 @@ function parseBlockFromDatabase(raw) {
 		version: parseInt(raw.version, 10),
 		timestamp: parseInt(raw.timestamp, 10),
 		height: parseInt(raw.height, 10),
-		previousBlock: raw.previousBlockId,
+		previousBlockId: raw.previousBlockId || '',
 		numberOfTransactions: parseInt(raw.numberOfTransactions, 10),
-		totalAmount: new BigNum(raw.totalAmount).toFixed(),
-		totalFee: new BigNum(raw.totalFee).toFixed(),
-		reward: new BigNum(raw.reward).toFixed(),
+		totalAmount: raw.totalAmount,
```

Comment:
I think, most of the stuff, if it's exactly the same you could do below, so that we don't need to update if there is no parsing required
```
const block = {
  ...raw,
  previousBlockId: raw.previousBlockId || '',
}
```